src/solver/conv_hip_implicit_gemm_v4r4_gen_xdlops_wrw_fp32.cpp

```diff
@@ -542,6 +544,15 @@ ConvSolution ConvHipImplicitGemmV4R4GenXdlopsWrWFp32::GetSolution(
     // clang-format on
 
     result.construction_params.push_back(construction_parameters);
+
+    result.invoker_factory = [](const std::vector<Kernel>& kernels) {
+        return [=](const Handle& handle, const boost::any& primitve_params) {
```

Comment:
SetTensor is missing for fp32. Please take a look at the deleted line 4212 in convolutionocl.cpp file in this PR